src/components/BookSearch/SelectReturnedBookModal.js

```diff
@@ -0,0 +1,53 @@
+import { Modal, Button } from "react-bootstrap";
+import styles from "../styles/bookItem.module.css";
+import UserBookManager from "../../common/UserBookManager";
+import ReturnBookItem from "./ReturnBookItem";
+
+export default function SelectReturnedBookModal({
+  onHide,
+  show,
+  selectedUserId,
+}) {
+  const userBookManager = new UserBookManager();
+  const borrowedBooks = userBookManager.getAllBorrowedBooks();
+
+  return (
+    <Modal
+      onClick={(e) => {
+        e.stopPropagation();
+      }}
```

Comment:
to chyba nie powinno nic zmieniać, te modale są w home page bezpośrednio więc sprawdz czy wywalenie tego coś zmienia ale nie powinno